Mathlib/Topology/Algebra/PontryaginDual.lean

```diff
@@ -0,0 +1,102 @@
+/-
+Copyright (c) 2022 Thomas Browning. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Thomas Browning
+-/
+import Mathlib.Analysis.Complex.Circle
+import Mathlib.Topology.Algebra.ContinuousMonoidHom
+
+#align_import topology.algebra.continuous_monoid_hom from "leanprover-community/mathlib"@"6ca1a09bc9aa75824bf97388c9e3b441fc4ccf3f"
+
+/-!
+# Pontryagin dual
+
+This file defines the Pontryagin dual of a topological group.
```

Comment:
Could you please expand the module docstring a bit? E.g., add some external references and/or examples. Otherwise LGTM. Thanks!

bors d+